frame_id: "arm_joint_left"
sensor_name: "arm_joint_left"
device_ID: 227 # this means "E3"
# pause_time: 8000
can_sub_topic: "/from_can_bus"
pub_topic_can: "/to_can_bus"

tare: false # if false the three below do nothing
roll_granularity_deg: 90
pitch_granularity_deg: 90
yaw_direction_deg: 0.0

set_orientation: false # if false the three below do nothing
x_rotation: 0.0
y_rotation: 0.0
z_rotation: 0.0

reset_attitude: false

set_new_source_address: true
new_source_address: 228